### PR TITLE
Add Negative Scenario for Mirror Metrics

### PR DESCRIPTION
**Pull request by hkadam134**
Adding a new Scenario to the existing Test Case : 
No new Test Suite is required for these changes.

[CEPH-83584059 - Validate end-to-end cephfs-mirroring metrics reports](https://polarion.engineering.redhat.com/polarion/redirect/project/CEPH/workitem?id=CEPH-83584059)

Logs: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-F73NJZ/

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_mirroring/test_cephfs_mirroring_metrics.py [hunk 1]
```diff
@@ -528,6 +528,72 @@ def run(ceph_cluster, **kw):
...
+            target_clients[0],
+            target_mount_path1,
+            target_client_user,
+            source_path1,
+            snap_target,
+            target_fs_new,
+        )
+
+        log.info(
+            "Wait some some time and check if the counters are reflecting the sync failure"
```
Comment (sumabai) [inline tests/cephfs/cephfs_mirroring/test_cephfs_mirroring_metrics.py:549]: typo on 'some'

**Review by sumabai — APPROVED** [review]
lgtm

File: tests/cephfs/cephfs_mirroring/test_cephfs_mirroring_metrics.py (hunk 1 above)
Comment (hkadam134) [reply tests/cephfs/cephfs_mirroring/test_cephfs_mirroring_metrics.py]: Fixed

**Review by sumabai — APPROVED** [review]
lgtm

**Review by neha-gangadhar — APPROVED** [review]
lgtm

**Pull request merged**